### ISSUE TITLE
MetricsGrabber test can't connect to Kubelet in GKE

### ISSUE THREAD

**Issue by gmarek** [kubernetes/kubernetes#19468]
http://kubekins.dls.corp.google.com/view/Critical%20Builds/job/kubernetes-e2e-gke-ci/10314/
```
02:14:42   Expected error:
02:14:42       <*errors.StatusError | 0xc20938f580>: {
02:14:42           ErrStatus: {
02:14:42               TypeMeta: {Kind: "", APIVersion: ""},
02:14:42               ListMeta: {SelfLink: "", ResourceVersion: ""},
02:14:42               Status: "Failure",
02:14:42               Message: "an error on the server has prevented the request from succeeding (get nodes gke-jkns-gke-e2e-ci-b4dff449-node-g54r:10250)",
02:14:42               Reason: "InternalError",
02:14:42               Details: {
02:14:42                   Name: "gke-jkns-gke-e2e-ci-b4dff449-node-g54r:10250",
02:14:42                   Group: "",
02:14:42                   Kind: "nodes",
02:14:42                   Causes: [
02:14:42                       {
02:14:42                           Type: "UnexpectedServerResponse",
02:14:42                           Message: "Error: 'use of closed network connection'\nTrying to reach: 'https://10.240.0.4:10250/metrics'",
02:14:42                           Field: "",
02:14:42                       },
02:14:42                   ],
02:14:42                   RetryAfterSeconds: 0,
02:14:42               },
02:14:42               Code: 503,
02:14:42           },
02:14:42       }
02:14:42       an error on the server has prevented the request from succeeding (get nodes gke-jkns-gke-e2e-ci-b4dff449-node-g54r:10250)
02:14:42   not to have occurred
```

It doesn't happen in GCE runs. 

I disabled the test for GKE, but we should find out why we can't proxy to nodes, i.e. why
```
rawOutput, err := g.client.Get().
		Prefix("proxy").
		Resource("nodes").
		Name(fmt.Sprintf("%v:%v", nodeName, kubeletPort)).
		Suffix("metrics").
		Do().Raw()
```
does not succeed. @kubernetes/goog-gke

**Comment by piosz** [comment]
please revert #19469 once the issue is fixed

**Comment by roberthbailey** [comment]
It's possible that #19314 may help with this.

**Comment by gmarek** [comment]
Ping me when it's merged - we'll give it a try.

**Comment by cjcullen** [comment]
In the failed run linked above, the MetricsGrabber test runs right after a node reboot test, so it is almost certainly a dangling tunnel. #19314 will make this better.

**Comment by cjcullen** [comment]
#19314 was merged last week.

**Comment by gmarek** [comment]
Fixed by #21003
